### ISSUE TITLE
erlang 20 eunit failure

### ISSUE THREAD

**Issue by mathiasertl** [basho/riak_pb#226]
When I run the eunit tests (e.g. using `make test`), I get the following error:

```
==> erlang-riak-protobuf (eunit)
Compiled src/riak_pb_kv_codec.erl
Compiled src/riak_pb_dt_codec.erl
src/riak_pb_codec.erl:33: export_all flag enabled - all functions will be exported
Compiling src/riak_pb_codec.erl failed:
ERROR: eunit failed while processing /home/mati/repositories/erlang-riak-protobuf: rebar_abort
tools.mk:38: recipe for target 'test' failed
make: *** [test] Error 1
```

This only occurs with Erlang 20. Googling the error yields some similar issues (e.g. [this one](https://github.com/erlang/rebar3/issues/1414)) that seem to have it solved already.

**Comment by martincox** [comment]
It's because compiler option ```warn_export_all``` became enabled by default in OTP20. Adding ```-compile(nowarn_export_all)``` will suppress the errors, which I guess is fine for use in test modules or ifdef test blocks.

**Comment by martincox** [comment]
Fixed by #227

**Comment by martincox** [comment]
Has been upgraded to OTP20/rebar3 in develop-3.0 bf5449e1f660c83041c73387b4a090e8b92da6c1
